platform/v1alpha1/platformoperators.crd.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
```

Comment:
This looks like a techpreview API.  TechPreview CRDs are not supposed to be installed by default.  Is your operator managing and gating the installation or using the CVO?  if the CVO, you need to add this annotation

```
    release.openshift.io/feature-set: TechPreviewNoUpgrade
```

Even if you're managing the installation, I'd still suggest the annotation to be clear about the intent.

Reply:
Yep, updated in the latest force push. We're planning on removing the hacks in the o/platform-operators repository once the requisite cluster profile annotations have landed here.